File: Javascript/level 2/Create Strange Characters.js
function toWeirdCase(s){
  var result = ""
  //함수를 완성해주세요
  var list = s.split(" ");
  for(var a=0;a<list.length;a++){
    var tem =list[a].split("");
  	for(var b=0; b<tem.length;b++){
     	if(b%2==1)
        result+=tem[b].toLowerCase();
   		else
        result+=tem[b].toUpperCase();
    }
    if(a<list.length-1)
    	result+=" ";
  }
  return result;
}


// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log("결과 : " + toWeirdCase("try hello world"));

File: Python/level 2/Create Strange Characters.py
def toWeirdCase(s):
    # 함수를 완성하세요
    list = s.split(" ")
    list2 = []
    for str in list:
        answer=""
        for tst in range(len(str)):
            if tst %2==0:
                answer+=str[tst].upper()
            else:
                answer+=str[tst].lower()
        list2.append(answer)
    
    return " ".join(list2)

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print("결과 : {}".format(toWeirdCase("try hello world")));